corehq/apps/api/resources/v0_5.py

```diff
@@ -263,13 +264,15 @@ def obj_create(self, bundle, request=None, **kwargs):
             bundle.obj.save()
         except Exception:
             if bundle.obj._id:
-                bundle.obj.retire()
+                bundle.obj.retire(deleted_by=request.user, deleted_via=USER_CHANGE_VIA_API)
             try:
                 django_user = bundle.obj.get_django_user()
             except User.DoesNotExist:
                 pass
             else:
-                django_user.delete()
+                django_user.delete(deleted_by=request.user, deleted_via=USER_CHANGE_VIA_API)
```

Comment:
does the django user model accept these arguments?

Reply:
that is incorrect, I will fix that. Thanks for catching it